app/assets/stylesheets/dlme.scss

```diff
@@ -169,7 +169,11 @@ body {
 }
 
 .masthead .site-title {
-  white-space: normal;
+  visibility: hidden;
+
+  + small {
```

Comment:
What is the purpose of this?  What content is in the small that we want hidden?  Can we just put the hidden style on that element in the template so it's easier to understand the connection?

Reply:
Spotlight by default inserts the exhibit title and some other content, but we don't want to show it because DLME has only one exhibit and the title is rendered elsewhere. I think this rendering happens in spotlight, so we'd need to override an entire template to do that. Part of the point of this PR was reducing the amount of things we override.

Reply:
Added a comment to note the above